src/qml/components/AttachmentPreview.qml

```diff
@@ -1,16 +1,21 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.12
 
+import "../theme"
+
 Item {
     id: attachmentPreview
+    onVisibleChanged: startAnimation.restart()
```

Comment:
Animation name is "start" but it also restarted at the end when visibility set to false. Should you call it for visible = true only?

Reply:
It will be rendered only once, when VISIBLE === true